test/unit/query-interface/bulk-insert.test.js

```diff
@@ -13,7 +14,7 @@ describe('QueryInterface#bulkInsert', () => {
   });
 
   // you'll find more replacement tests in query-generator tests
-  it('does not parse replacements outside of raw sql', async () => {
+  (dialect !== 'oracle' ? it : it.skip)('does not parse replacements outside of raw sql', async () => {
```

Comment:
What is the reason for disabling this test? It's an important one to prevent sql injections

Reply:
Bulk insert for Oracle uses binds, need to explore this a bit for enabling this test.

Reply:
Are you going to work on this?

Reply:
We don't support replacements in BulkInsert, the only way to do bulk insert in the Oracle dialect is using binds. So using replacement while doing bulk insert would generate an error.